dvc/command/daemon.py

```diff
@@ -12,11 +12,14 @@ class CmdDaemonUpdater(CmdDaemonBase):
     def run(self):
         import os
         from dvc.repo import Repo
+        from dvc.config import Config
         from dvc.updater import Updater
 
         root_dir = Repo.find_root()
         dvc_dir = os.path.join(root_dir, Repo.DVC_DIR)
-        updater = Updater(dvc_dir)
+        config = Config(dvc_dir, verify=False)
```

Comment:
Why do you use unvalidated config?

Reply:
@Suor because it shouldn't fail if it doesn't understand the config. Same in analytics and friends.

Reply:
@efiop does config have `verify` argument? When I run any `dvc` command, I get `ERROR: unexpected error - __init__() got an unexpected keyword argument '
verify'` in the log.

Reply:
@pared Good catch! Yes, it should be `validate`. My bad.